### DISCUSSION TITLE
What exactly is a repository?

### DISCUSSION THREAD

**Discussion by 12hrg** [community/community#49561] [New to GitHub]
I am new to github and I wrote some stuff but it disappeared and I wanted to know what happened to it and what exactly is a repository if someone can answer my questions please 🙏

**Comment by 12hrg** [comment]
Maybe my writing got lost because I didn't know how to save it?

**Comment by evahaudi** [comment]
what is the easiest way to upade a repository accss token,i updated one but it still requesting for password when i git push

**Comment by 03timothyash** [comment]
Así es . Así esta bien todo

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by enviGit** [comment]
### Hello Herman!

So, first of all **a repository**, often shortened to "**repo**", is a location where code is stored and managed. It's like a folder that holds all the files and folders that make up a project, and it includes version control tools to track changes to the code over time.

- You may have accidentally deleted or overwritten the files. If you were working on your local machine, it's possible that you accidentally deleted or overwrote the files before committing them to your repository. If this is the case, the files are likely lost and cannot be recovered.
- You may have committed the files to the wrong branch or repository. If you're not familiar with how Git works, it's possible that you committed the files to the wrong branch or repository. If this is the case, you can try to locate the files in the other branch or repository.
- There may have been a problem with the repository itself. It's possible that there was an issue with the repository, such as a corruption or a problem with the hosting service. If this is the case, you should contact the hosting service to see if they can help recover the files.

As for what happens if a repository disappears, it depends on whether you have a local copy of the repository or not. If you do have a local copy, you can still access and work on the files. However, if you don't have a local copy, the files will be lost and cannot be recovered unless you have a backup. This is why it's important to regularly back up your work, either by making copies of the files or by using a cloud backup service.

**Comment by 03timothyash** [comment]
Deudas de. Pagó.

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Ntshangase** [comment]
Hi, To learn more about repositories you can check out: https://docs.github.com/en/repositories/creating-and-managing-repositories/about-repositories
